### PR TITLE
UI cleanup + add radium

### PR DESCRIPTION
**Pull request by madelynkasula**

### CODE REVIEW

File: src/demo/ui.jsx [hunk 1]
```diff
@@ -46,17 +39,14 @@ const styles = {
     backgroundColor: colors.white,
     fontSize: '120%',
     borderRadius: 8,
-    border: `2px solid ${colors.black}`,
     minWidth: 160,
     padding: '16px 30px',
-    ':focus': {
```
Comment (madelynkasula) [inline src/demo/ui.jsx]: this didn't work when i added it in #130 😢but should work once Radium is actually being used

Comment (breville) [reply src/demo/ui.jsx]: Doesn't this PR start it being used?

File: src/demo/ui.jsx [hunk 2]
```diff
@@ -432,6 +410,19 @@ class SpeechBubble extends React.Component {
 }
 
 class ActivityIntro extends React.Component {
+  onClickContinue = () => {
+    const state = getState();
+    if (state.loadTrashImages) {
+      setState({
+        currentMode: Modes.TrainingIntro,
```
Comment (breville) [inline src/demo/ui.jsx]: Should we have all transitions to a new `currentMode` go through our `toMode` helper?  Amongst other things, it seems to init the model.

File: src/demo/ui.jsx (hunk 1 above)
Comment (madelynkasula) [reply src/demo/ui.jsx]: i installed Radium, but haven't actually set it up yet. i'll do that in a follow-up PR to keep this one small-ish

**Pull request merged**